payment-gateway/go-example/main.go

```diff
@@ -196,8 +197,11 @@ func userDoGaslessPayment(receiver common.Address, amount *big.Int) {
 	fmt.Printf("Sponsorable Information:\n%+v\n", sponsorableInfo)
 
 	if sponsorableInfo.Sponsorable {
-		// Send the transaction using PaymasterClient
-		_, err := paymasterClient.SendRawTransaction(context.Background(), txInput)
+		// We strongly encourage you to set the UserAgent value. It should represent
+		// your wallet name or brand name. This information is for further statistical
+		// analysis and insight. Setting a unique UserAgent will help MegaFuel to
+		// better understand wallet usage patterns and improve service.
+		_, err = paymasterClient.SendRawTransaction(context.Background(), txInput, &paymasterclient.TransactionOptions{UserAgent: "myWalletName/v1.0.0"})
```

Comment:
`paymasterclient.TransactionOptions`, even though we use the `UserAgent`, but can we provide a better interface for user to understand, like `&paymasterclient.TransactionOptions{WalletToTrace: "myWalletName/v1.0.0"}`